packages/web-components/tsconfig.json

```diff
@@ -1,11 +1,11 @@
 {
   "extends": "../../tsconfig.base.wc.json",
   "compilerOptions": {
-    "target": "ES2019",
-    "module": "ESNext",
     "experimentalDecorators": true,
     "resolveJsonModule": true,
-    "allowJs": true
+    "allowJs": true,
+    "lib": ["ESNext", "DOM"],
+    "types": ["web", "es2023-shim"]
```

Comment:
TS solution configs shouldn't contain types definition as they make sense only on environment scoped level ( which is .lib.json, '.spec.json', 'storybook', etc ).

also this kind of changes leads to inconsistencies which can be observed in this PR, storybook tsconfig provides types ( this is correct ), while others are not.

please revert this change

Reply:
> this kind of changes leads to inconsistencies which can be observed in this PR, storybook tsconfig provides types ( this is correct ), while others are not.

The `types` property is inherited from the use of `extends`. The storybook tsconfig sets its own `types` property to override the defaults provided here (to add `node` types). Is there any reason not to use this built-in pattern?

https://www.typescriptlang.org/tsconfig/#extends
> The configuration from the base file are loaded first, then overridden by those in the inheriting config file.

Reply:
I understand what you wanna do,

Wondering if you read my message ? 

> TS solution configs shouldn't contain types definition as they make sense only on environment scoped level ( which is .lib.json, '.spec.json', 'storybook', etc ). + following monorepo consistency patterns.  

ty